api/models.py

```diff
@@ -168,6 +168,7 @@ class User(AbstractUser):
     picture = models.CharField(max_length=255, blank=True, null=True)
     phone_number = models.CharField(max_length=50, blank=True, null=True)
     last_modified = models.DateTimeField(auto_now=True, editable=False)
+    password = models.CharField(max_length=128, blank=True, default='')
```

Comment:
we could do `null=True` and get rid of the default

Reply:
on it